chirpus-two/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="nav">
+        <h5>chirpus inc.</h5>
+        <a href="#" class="nav__link">Work</class=></a>
```

Comment:
I would put lines 14-17 into their own parent div, then in CSS give .nav the property justify-content: space-between; to use a flex property to separate the navigation items.